src/main/java/com/WebController/EventController.java

```diff
@@ -54,6 +68,10 @@ public String receivePatient (@RequestBody List<Event> events){
         return "success";
     }
 
+    /**
+     * DoGet handler for all events.
```

Comment:
this could get very long, and so getting Paged answers might be the way to go. Also, I don't think you need to collect everything into a list necessary. I think you can just return the Iterable, and Spring will serialize the return for you. The advantage of the Iterable could be that especially in combination with the Paging you may be able to avoid retrieving the entire table if it's not necessary.